contrib/local-environment/traefik/traefik.yaml

```diff
@@ -0,0 +1,7 @@
+api:
+  insecure: true
+log:
+  level: DEBUG
```

Comment:
Do we want to keep this as debug? I had a play with the environment and the logs were BIG! :sweat_smile: I think info may be better and then users can tweak as appropriate right?